docker/docker-compose.yml

```diff
@@ -179,6 +179,41 @@ services:
       localstack:
         condition: service_healthy
 
+  management-ui:
+    healthcheck:
+      test: ["CMD", "wget", "-qo", "/dev/null", "http://localhost:8099/health"]
+      interval: 10s
+      timeout: 10s
+      retries: 3
+    image: quay.io/ukhomeofficedigital/hocs-management-ui:${MANAGEMENT_TAG:-latest}
+    ports:
+      - 8099:8099
```

Comment:
Should this map to `8080`?

Reply:
Clashes with frontend as that uses 8080. As the management-ui runs on 8099 internally pointing it to 8099 on the container hits it.

Reply:
This has been resolved, no points to 8080 internally and 8099 to the user.